### ISSUE TITLE
iOS • CSS • Examples Gallery elements stretched full width

### ISSUE THREAD

**Issue by fffiloni** [gradio-app/gradio#7385]
### Describe the bug

Hello ! 

On Safari iOS, the 4.* gradio Examples component gallery displays img elements stretched full-width:
![IMG_1252](https://github.com/gradio-app/gradio/assets/7447007/8ed6d0cf-f64c-4f6c-87ff-e66727b47f7a)

**Temporary fix** with custom CSS in app.py, where "component-11" is the example component div:

```css
/* fix examples gallery width on mobile */

div#component-11 > .gallery > .gallery-item > .container > img {
    width: auto!important;
}
```

Instead of `width: 100%;`

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

Try the Examples component with several image and open it on iOS Safari

### System Info

```shell
Gradio 4.*
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Hi @fffiloni can you share a simple repro for us to reproduce the issue?

**Comment by fffiloni** [comment]
sure, you can experience ot on going to this demo on iOS Safari :

https://huggingface.co/spaces/fffiloni/lcm-img-variations

i did not add the custom css yet to fix the described behavior. 
![IMG_1253](https://github.com/gradio-app/gradio/assets/7447007/0dd47133-c5e9-414e-b074-4e0165a56d9e)

**Comment by pngwn** [comment]
Ah we missed this from the changelog:

```md
- `gr.Gallery` will now convert all images to panoramas!
```

**Comment by abidlabs** [comment]
@fffiloni just to confirm, you're using the `gr.Image()` component here, not `gr.Gallery()`, right? So the examples for `gr.Image()` are broken?

**Comment by fffiloni** [comment]
@abidlabs No, there is a misunderstanding: this behavior happens to the **gr.Examples** component, when you only provide list of images as example inputs

**Comment by fffiloni** [comment]
Seems to be fixed in 4.18.0 🤗 
I close the issue, sorry if i wasn’t clear enough on first message 🙏
